robolectric-shadows/shadows-core/src/main/resources/org/robolectric/shadows/ShadowAssetManager.java.vm

```diff
@@ -270,6 +272,15 @@ public final class ShadowAssetManager {
     return new String[0]; // todo
   }
 
+  /**
+   * Creates a {@link RoboAttributeSet} for the given {@link Attribute}(s)
+   */
+  public RoboAttributeSet createAttributeSet(Attribute... attrs) {
```

Comment:
As discussed, I think this might work better as a static factory method taking Context as a method parameter. It will make calling code cleaner. I think it can still live on this class. We can also make the return value simply AttributeSet thus not leaking the RoboAttributeSet. It only has a couple of methods like put() on it, but I'm not sure of the value of having the attribute set mutable, so perhaps we can remove these?